resources/views/components/threads/reply.blade.php

```diff
@@ -31,6 +31,14 @@ class="prose prose-lio max-w-none p-6 break-words"
     >
     </div>
 
+    @if($reply->updatedBy())
+        <div class="text-sm text-gray-900 p-6">
+            Last edit by
+            <a href="{{ route('profile', $reply->updatedBy()->username()) }}" class="text-lio-500 border-b-2 pb-0.5 border-lio-100 hover:text-lio-600">{{ $reply->updatedBy()->name() }}</a>
```

Comment:
```suggestion
            <a href="{{ route('profile', $reply->updatedBy()->username()) }}" class="text-lio-500 border-b-2 pb-0.5 border-lio-100 hover:text-lio-600">{{ '@'.$reply->updatedBy()->name() }}</a>
```

Reply:
The result will be `Last edit by @Sanjit Singh on 12 July 2021` which looks bit off to me.

@driesvints thoughts?